src/RevaSadovnicova/Main.java

```diff
@@ -0,0 +1,68 @@
+package RevaSadovnicova;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+       //int x=4;
+       //int y=6;
+        //if(x>y){
+          //  System.out.println("no");
+       // }
+       // else if (x<y){
+           // System.out.println("no");
+        //}
+       // else{
+           // System.out.println("yes");
+       // }
+        //int x=4;
+       // int y=7;
+       // int sum=x+y;{
+           // System.out.println("sum");
+        //}
+     //int x=30;
+     //if(x>30){
+        // System.out.println("negative");
+     //}
+     //else if (x<30){
+            //System.out.println("positive");
+        //}
+    // else {
+        // System.out.println("is zero");
+    // }
+        //int x = 17;
+        //int y =5;
+        //if (x>y){
+           // System.out.println("x больше y");
+        //} else if (x<y) {
+            //System.out.println("y больше x");
+
+        //}
+        //else {
+           // System.out.println("равно");
+        //}
+        //String string;
+        //Scanner sc = new Scanner(System.in);
+        //System.out.println("Enter name: ");
+        //string = sc.nextLine();
+        //System.out.println("name is : " + string.length());
+
+
+    }
+    }
+
+
+// {
+    //public static void main(String[] args) {
+       // int number = 10;
+
+        //if (number > 0) {
+        //    System.out.println(" number is positiv");
+       // } else if (number < 0) {
+          //  System.out.println("number is negativ");
+       // } else {
+          //  System.out.println("number is zero");
+        //}
+   // }
+//}
```

Comment:
You did not closed } } 
It should be fixed, otherwise syntax error.

Reply:
Yes I did Line 51, 52)))